tests/sentry/api/endpoints/test_relay_globalconfig.py

```diff
@@ -0,0 +1,878 @@
+from uuid import uuid4
+
+import pytest
+from django.urls import reverse
+from sentry_relay.auth import generate_key_pair
+
+from sentry.models.relay import Relay
+from sentry.utils import json
+from sentry.utils.pytest.fixtures import django_db_all
+
+
+@pytest.fixture
+def key_pair():
+    return generate_key_pair()
+
+
+@pytest.fixture
+def public_key(key_pair):
+    return key_pair[1]
+
+
+@pytest.fixture
+def private_key(key_pair):
+    return key_pair[0]
+
+
+@pytest.fixture
+def relay_id():
+    return str(uuid4())
+
+
+@pytest.fixture
+def relay(relay_id, public_key):
+    return Relay.objects.create(relay_id=relay_id, public_key=str(public_key), is_internal=True)
```

Comment:
Let's import these fixtures instead of redefining them.

Reply:
by adding a `conftest.py` file? ran into some errors iwth that, also wondering why it's not done anywhere else. seems the convention is to redefine these when i look at the other tests

Reply:
Importing works for me:
```suggestion
from .test_relay_projectconfigs_v3 import key_pair, public_key, private_key, relay_id, relay
```

Reply:
the tests passed like this? for me they fail